Podman Desktop Goax E2E is sometimes unstable and cannot start up podman-desktop

```
[BeforeSuite] 
D:/a/podman-desktop-e2e/podman-desktop-e2e/test/e2e/e2e_podman/suite_test.go:84
  [FAILED] in [BeforeSuite] - D:/a/podman-desktop-e2e/podman-desktop-e2e/test/e2e/e2e_podman/suite_test.go:96 @ 08/07/24 10:02:30.257
[BeforeSuite] [FAILED] [43.722 seconds]
[BeforeSuite] 
D:/a/podman-desktop-e2e/podman-desktop-e2e/test/e2e/e2e_podman/suite_test.go:84

  [FAILED] Unexpected error:
      <*errors.errorString | 0xc000292230>: 
      error disabling telemetry :can not find any element with id Enable telemetry
      {
          s: "error disabling telemetry :can not find any element with id Enable telemetry",
      }
  occurred
  In [BeforeSuite] at: D:/a/podman-desktop-e2e/podman-desktop-e2e/test/e2e/e2e_podman/suite_test.go:96 @ 08/07/24 10:02:30.257
------------------------------
[AfterSuite] 
D:/a/podman-desktop-e2e/podman-desktop-e2e/test/e2e/e2e_podman/suite_test.go:103
[AfterSuite] PASSED [0.000 seconds]
------------------------------
[ReportAfterSuite] Podman Desktop e2e JUnit report
D:/a/podman-desktop-e2e/podman-desktop-e2e/test/e2e/e2e_podman/suite_test.go:57
[ReportAfterSuite] PASSED [0.001 seconds]
------------------------------

Summarizing 1 Failure:
  [FAIL] [BeforeSuite] 
  D:/a/podman-desktop-e2e/podman-desktop-e2e/test/e2e/e2e_podman/suite_test.go:96

Ran 0 of 2 Specs in 43.722 seconds
FAIL! -- A BeforeSuite node failed so all tests were skipped.
--- FAIL: TestE2E (43.72s)
FAIL
Remove-Item: C:\Users\rhqp\pd-e2e\run.ps1:61
Line |
  61 |  rm junit_report.xml
     |  ~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\Users\rhqp\junit_report.xml' because it does not exist.
Get-Process: C:\Users\rhqp\pd-e2e\run.ps1:66
Line |
  66 |  $pdpid=Get-Process "Podman Desktop" | Select Id -ExpandProperty Id |  .
     |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find a process with the name "Podman Desktop". Verify the process name and call the cmdlet again.
Stop-Process: C:\Users\rhqp\pd-e2e\run.ps1:67
Line |
  67 |  Stop-Process -ID $pdpid -Force
     |                   ~~~~~~
     | Cannot bind argument to parameter 'Id' because it is null.
```

Referenced run: https://github.com/containers/podman-desktop-e2e/actions/runs/10281807672/job/28452124492

--- a/test/extended/podman-desktop/podman-desktop.go
+++ b/test/extended/podman-desktop/podman-desktop.go
@@ -33,7 +33,7 @@ func Open(execPath string) (*PDApp, error) {
 		return nil, fmt.Errorf("error opening the podman desktop executable at %s: %v", execPath, err)
 	}
 	// We open remotely so we wait for a bit
-	delay.Delay(delay.LONG)
+	delay.Delay(delay.XLONG)
 	if context.SaveScreenshots() {
 		if err := screenshot.CaptureScreen(context.TestContext.ScreenshotsOutputPath, "openApp"); err != nil {
 			logging.Errorf("error capturing the screenshot: %v", err)

